app/controllers/partners_api/application_controller.rb

```diff
@@ -0,0 +1,7 @@
+require_dependency 'api/entities_helpers'
+
+class PartnersApi::ApplicationController < ActionController::Base
+  Grape::Entity.include Api::UrlHelpers
```

Comment:
I imagine this is being done somewhere else. If you need to do this in different places, I'd say it's best to do this in an `config/initializer`